server.py

```diff
@@ -0,0 +1,108 @@
+from tensorflow.contrib.framework.python.ops import audio_ops
+import base64
+import cherrypy
+import json
+import io
+import numpy as np
+import os
+import tensorflow as tf
+import wave
+import zlib
+
+class LabelService(object):
+    def __init__(self, graph_filename, labels=["_silence_", "_unknown_", "anserini", "random"]):
+        with tf.gfile.FastGFile(graph_filename, "rb") as f:
+            graph_def = tf.GraphDef()
+            graph_def.ParseFromString(f.read())
+            tf.import_graph_def(graph_def, name="")
+        self.labels = labels
+        self.sess = tf.Session()
+
+    def label(self, wav_data):
+        """Labels audio data as one of the specified trained labels
+
+        Args:
+            wav_data: The WAVE to label
+
+        Returns:
+            A (most likely label, probability) tuple
+        """
+        output = self.sess.graph.get_tensor_by_name("labels_softmax:0")
+        predictions, = self.sess.run(output, {"wav_data:0": wav_data})
+        return (self.labels[np.argmax(predictions)], max(predictions))
+
+def json_in(f):
```

Comment:
4 spaces for indent

Reply:
Oh oops, I copied this from one of my older projects...